### PR TITLE
Allow passing of arbitrary options to node-notifier

### PR DESCRIPTION
**Pull request by JFreemanActiveCampaign**
node-notifier has lots of wonderful options we can change, so we should allow people to use them. This allows, for example, enabling of sounds, custom images and clickable notifications

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -11,15 +11,15 @@ module.exports = {
...
+      notifier.buildError(error, config.buildError.options);
     }
   },
 
   postBuild: function(results) {
     var config = Config.load(this.project.root);
 
     if (config.buildSuccess.notify) {
-      notifier.buildSuccess(results);
+      notifier.buildSuccess(results, config.buildSuccess.options);
```
Comment (pdud) [inline index.js]: Awesome PR – thank you!!!

Should these be `notifier.buildSuccess(results, config.buildSuccess)` and `notifier.buildError(error, config.buildError)`?

Comment (JFreemanActiveCampaign) [inline index.js]: Yes they should be! Good catch.

It's an awesome package, I've been looking for something like this for a while, and thought I would have to roll my own

**Comment by pdud** [comment]
Thanks you very much! I'll get a new version on npm.

**Comment by JFreemanActiveCampaign** [comment]
Thank you!

**Pull request merged**